modules/ROOT/pages/push/provisioning.adoc

```diff
@@ -1,6 +1,6 @@
 include::{partialsdir}/attributes.adoc[]
 
-:service-name: Push Notifications
+:service-name: Unified Push Server
```

Comment:
what's this? should be same as attributes.adoc

Reply:
I changed it to `Unified Push Server` because thats the name you see in the service catalog. It would be confusing for the provisioning instructions to call it `Push Notification Service` when the name of the APB actually is `Unified Push Server`.

Or should we change the APB name / change the service name in attributes.adoc?

Reply:
Peter you have a point here. IMO the confusion comes with the name `Unified Push Server`, it's inaccurate and I tend to associate it to the UI only. It would make more sense to find it as **Push Notifications** or simply **Push** in the catalog. Then, I propose to leave it here as `Push Notifications` for now and try to change APB's name.

Reply:
@pb82  Name needs to be changed in service catalog -ping @maleck13